atmo/atmo_test.go

```diff
@@ -0,0 +1,102 @@
+package atmo
+
+import (
+	"bytes"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/suborbital/atmo/atmo/options"
+	"github.com/suborbital/vektor/vtest"
+)
+
+func TestHelloEndpoint(t *testing.T) {
+	atmo := atmoForBundle("../example-project/runnables.wasm.zip")
+
+	server, err := atmo.testServer()
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	vt := vtest.New(server) //creating fake version of the server that we can send requests to and it will behave same was as if it was the real server
+
+	req, _ := http.NewRequest(http.MethodPost, "/hello", bytes.NewBuffer([]byte("my friend"))) //same way to use curl
+
+	resp := vt.Do(req, t) //execute the request and returns the response  (can use resp to do the checks)
+
+	resp.AssertStatus(200) //checks if response status code is 200 and if not it will fail
+	resp.AssertBodyString("hello my friend")
```

Comment:
Minor thing, this can be simplified with this call chaining pattern, since it returns a reference to itself. They're pretty much the same though.

```suggestion
	vt.Do(req, t).
		AssertStatus(200).
		AssertBodyString("hello my friend")
```

Reply:
Love it!! Didn't know that was an option, makes it more dry.